tests/PKSim.Tests/Presentation/CreatePartialProcessPresenterSpecs.cs

```diff
@@ -83,5 +92,11 @@ public void should_not_reset_the_name_and_protein_entered_by_the_user()
          _process2DTO.MoleculeName.ShouldBeEqualTo("Protein");
          _process2DTO.DataSource.ShouldBeEqualTo("Lab");
       }
+      
+      [Observation]
+      public void should_not_reset_the_name_of_the_metabolite()
+      {
+         _enzymaticPartialProcessDTO.Metabolite.ShouldBeEqualTo(_template1.MetaboliteName);
```

Comment:
That's the test here. Before the Metabolite was empty. With the implementation, the Metabolite is whatever was set before the update